docs/profiling.md

```diff
@@ -0,0 +1,92 @@
+# vHive profiling tool
+
```

Comment:
before the methodology, you should explain what purpose the tool serves. which research goals it may help? what is the motivation for it?

Reply:
Done